src/components/Editor/index.jsx

```diff
@@ -38,7 +38,11 @@ const Editor = ({ getNewSlide, deleteSlide, activePresentation }) => {
   }
 
   const addText = () => {
-    canvas.add(new fabric.IText('Edit Me'))
+    let text = new fabric.IText('Edit Me')
+    text.on('dblclick', function(e) {
```

Comment:
what is the purpose of the double click handler?

Reply:
testing, its removed in the next push